### PR TITLE
Feat slim 605

### PR DESCRIPTION
**Pull request by bjornamkreutz**
SLIM-605

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Integration-Tests?pullRequest=162) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: cucumber-tests-platform-dlms/src/test/java/com/alliander/osgp/cucumber/platform/dlms/Keys.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Smart Society Services B.V.
+ * Copyright 2017 Smart Society Services B.V.
```
Comment (rudidevries) [inline cucumber-tests-platform-dlms/src/test/java/com/alliander/osgp/cucumber/platform/dlms/Keys.java]: You shouldn't change the date of existing files. You could modify it like '2016-2017' to reflect changes are made over multiple years.

Comment (bjornamkreutz) [inline cucumber-tests-platform-dlms/src/test/java/com/alliander/osgp/cucumber/platform/dlms/Keys.java]: Done

**Comment by bjornamkreutz** [comment]
retest this please

**Comment by bjornamkreutz** [comment]
retest this please

**Pull request merged**